src/main.rs

```diff
@@ -321,8 +321,12 @@ fn write_profile(matches: &ArgMatches) -> Result<(), Box<dyn std::error::Error>>
 }
 
 fn run_rustpython(vm: &VirtualMachine, matches: &ArgMatches) -> PyResult<()> {
+    #[cfg(not(target_os = "wasi"))]
     import::init_importlib(&vm, true)?;
 
+    #[cfg(target_os = "wasi")]
+    import::init_importlib(&vm, false)?;
```

Comment:
You can use the `cfg` macro instead of the attribute here: `import::init_importlib(&vm, cfg!(not(target_os = "wasi")))`